tests/data_viz/test_share_link.py

```diff
@@ -46,6 +50,7 @@ def json():
 
     mocked.return_value = ResponsePost()
     mock_type.return_value = "test_machine"
+    mock_clean.return_value = ["test"]
```

Comment:
I am trying to understand why thre is not an additional verify step.  Assert X = y?

Reply:
We're asserting latter that the mock post is called with the correct state, where were using "machine_source": ["test"].  mock_clean and mock_type are intermediate calls gathering info from MA.